tldw_Server_API/app/core/Slides/style_packs/dashboard_glass.css

```
@@ -0,0 +1,15 @@
+.reveal[data-style-pack="dashboard_glass"] {
+  background-color: #0f172a;
+  background: linear-gradient(135deg, rgba(15, 23, 42, 0.98), rgba(30, 41, 59, 0.92));
```

Stylelint (17.5.0)

[error] Unexpected shorthand "background" after "background-color" (declaration-block-no-shorthand-property-overrides)

(declaration-block-no-shorthand-property-overrides)